test_data.py

```diff
@@ -18,7 +18,7 @@ class Usage(testdata.DictFactory):
 #xpected format: id:1|deviceId:4113947bec18b7ad|time:1481916197273|packageName:org.literacyapp|literacySkill:LETTER_IDENTIFICATION|numeracySkill:null|letter:a|number:null|word:null
 
 
-out = open("usage.txt", 'w')
+out = open("/home/bikash/repos/bigdata/DataGeneratorSimulator/usage.txt", 'w')
```

Comment:
Probably use an environment variable instead of an absolute path? Or get the absolute path programmatically. Or use a subfolder of this project